maro/rl/workflows/config/template.yml

```diff
@@ -2,38 +2,61 @@
 # Licensed under the MIT license.
 
 job: your_job_name
+# Path to a directory that defines a business scenario and contains the necessary components to execute reinforcement
+# learning workflows in single-threaded, multi-process and distributed modes.
 scenario_path: "/path/to/your/scenario"
-log_path: "/path/to/your/log/folder"
+log_path: "/path/to/your/log/folder"  # All logs are written to a single file for ease of viewing.
 main:
-  num_episodes: 100
-  num_steps: -1
+  num_episodes: 100  # number of episodes to run. Each episode is one cycle of roll-out and training.
+  # Number of environment steps to collect environment samples over. If null, samples are collected until the
+  # environments reach the terminal state, i.e., for a full episode. Otherwise, samples are collected until the
+  # specified number of steps or the terminal state is reached, whichever comes first.
+  num_steps: null
+  # This can be an integer or a list of integers. An integer indicates the interval at which policies are evaluated.
+  # A list indicates the episodes at the end of which policies are to be evaluated.
   eval_schedule: 10
-  logging:
-    stdout: INFO   # DEBUG, INFO, WARN, ERROR, CRITICAL, PROGRESS
-    file: DEBUG
+  logging:  # log levels for the main loop
+    stdout: INFO  # DEBUG, INFO, WARN, ERROR, CRITICAL, PROGRESS
+    file: DEBUG  # DEBUG, INFO, WARN, ERROR, CRITICAL, PROGRESS
 rollout:
+  # Optional section to specify roll-out parallelism settings. If absent, a single environment instance will be created
+  # locally for training and evaluation.
   parallelism:
-    sampling: 10
-    eval: 1  # defaults to 1 if not provided
-    min_env_samples: 8  # ignored if parallelism.training == 1
-    grace_factor: 0.2  # ignored if parallelism.training == 1
-    controller: # ignored if parallelism.training == 1
-      host: "127.0.0.1"   # ignored if run in containerized environments
-      port: 20000
-  logging:
-    stdout: INFO   # DEBUG, INFO, WARN, ERROR, CRITICAL, PROGRESS
-    file: DEBUG
+    sampling: 10  # Number of parallel roll-outs to collecting training data from.
+    eval: null  # Number of parallel roll-outs to evaluate policies on. Defaults to 1 if not provided.
```

Comment:
null means the field is optional. You can also comment it out or leave it blank. A note has been added at the top of the file.

Reply:
So if we use `eval: null`, it would be set as default value `1`?

Reply:
Yes, it will be set to 1 in the code. In other words, if you do not specify eval parallelism, only one roll-out worker will be used for evaluation by default.